src/io/memory.cr

```diff
@@ -336,6 +336,23 @@ class IO::Memory < IO
     @pos = value.to_i
   end
 
+  # Truncates the memory to the specified *size*.
+  #
+  # Note: a negative value truncates from the end of the `IO::Memory`.
```

Comment:
`Note` -> `NOTE`

Reply:
Its used in both ways throughout the project. I will use the caps as it doesn't really matter to me and seems more consistently applied.